slang.h

```diff
@@ -862,6 +862,7 @@ extern "C"
             SourceEmbedName,
             SourceEmbedLanguage,
             DisableShortCircuit,   // bool
+            MinimumSlangOptimization, // bool
```

Comment:
I'm wondering that if we should use -O<n> to indicate the level of optimization?

Reply:
For now I don't want to mix these two things up just yet. This allows `-O` to continue to affect downstream compilation without being tied to slang's logic.